benchmarks/inc/xoshiro.hpp

```diff
@@ -0,0 +1,39 @@
+/*  Written in 2018 by David Blackman and Sebastiano Vigna (vigna@acm.org)
+
+To the extent possible under law, the author has dedicated all copyright
+and related and neighboring rights to this software to the public domain
+worldwide. This software is distributed without any warranty.
+
+See <http://creativecommons.org/publicdomain/zero/1.0/>.

```

Comment:
We need to include this original license block (i.e., in the form it had before I added the SPDX-License-Identifier) in `NOTICES.txt`.